test/e2e/util.go

```diff
@@ -1484,22 +1484,22 @@ func validateController(c *client.Client, containerImage string, replicas int, c
 
 	By(fmt.Sprintf("waiting for all containers in %s pods to come up.", testname)) //testname should be selector
 waitLoop:
-	for start := time.Now(); time.Since(start) < podStartTimeout; time.Sleep(5 * time.Second) {
-		getPodsOutput := runKubectlOrDie("get", "pods", "-o", "template", getPodsTemplate, "--api-version=v1", "-l", testname, fmt.Sprintf("--namespace=%v", ns))
+	for start := time.Now(); time.Since(start) < PodStartTimeout; time.Sleep(5 * time.Second) {
```

Comment:
Why did this file change? It wasn't changed in the original PR and these changes seem unrelated to the rest of the cherry pick.